proc_scripts/load_data.py

```diff
@@ -209,27 +208,6 @@ def proc_DOSY_CPMG(s):
     # }}}
     return s
 
-def proc_90_pulse(s,fl=fl):
-    dw = diff(s.getaxis('t2')[0:2]).item()
-    # {{{ determine the phase corrections
-    s.ft('t2', shift=True)
-    s = ph1_real_Abs(s,dw)
-    # }}}
-    s_conv = s.C
-    s_conv.ift('t2')
-    s_conv *= exp(-8*pi*s_conv.fromaxis('t2'))
-    s_conv.ft('t2')
-    if fl is not None:
-        fl.next('raw - frequency domain')
-        fl.image(s)
-    if fl is not None:
-        fl.next('raw - time domain')
-        s.ift('t2')
-        fl.image(s)
-    return s
-
-
-
```

Comment:
again, the 90 pulse stuff belongs in a different pull request

Reply:
okay right now the 90 processing issue is associated with this pull request.

Reply:
no, issues and pull requests live independently of each other.  One issue can reference one or more pull requests, or vice versa.

Reply:
so, *e.g.*: you would just open a new branch off of master (or this branch, doesn't really matter), make a new pull request, then go make a comment in #24 where you reference that branch